src/app/map/components/map-controls/zoom-controls/zoom-controls.component.ts

```diff
@@ -15,6 +15,12 @@ import {selectMapUiState} from '../../../../state/map/reducers/map-ui.reducer';
 import {GeolocationState} from '../../../../state/map/states/geolocation.state';
 import {MapUiState} from '../../../../state/map/states/map-ui.state';
 
+const TOOLTIP_TEXT = {
```

Comment:
Same here.

Also, I think when we're going to use I18n, we'd have to refactor this probably anyway, and then we should ask ourselves whether we need this indirection.

(That is - if there is no specific reason for adding these tooltips as constant)

Reply:
see comment above